core/src/Modal.html

```diff
@@ -217,13 +291,34 @@ <h3 class="fd-dialog__title">{modalSettings.title}</h3>
     width: 100%;
     height: 100%;
   }
-  .fd-dialog {
-    &__content {
+
+  .modal-dialog__content{
       width: 80%;
```

Comment:
since you inject inline styling into the modal-dialog__content do we need this styling at all?

Reply:
Good point.. I will remove this class.